src/osd/SnapMapper.cc

```diff
@@ -440,24 +440,17 @@ void SnapMapper::add_oid(
   backend.set_keys(to_add, t);
 }
 
-int SnapMapper::get_next_objects_to_trim(
+void SnapMapper::get_objects_by_prefixes(
```

Comment:
if 'out' is an 'out only' parameter - return it from the function.
If it's an in-out - pls rename it, and clarify in the comment what might be there
when entering.

Reply:
The out param name was taken from existing code get_next_objects_to_trim() where it is used as an OUT parameter adding entries into a user supplied vector which in theory could have previous objects inside.
I don't see a reason to change the name

Reply:
There are two good reasons:
1) if isn't clear whether it's out or in/out, esp since the old name was 'out' for 'in-out', and
2) a clearer API is to return a created vector.